### DISCUSSION TITLE
Undefined Image ID when running Auto Segmentation on OHIF and it fails

### DISCUSSION THREAD

**Discussion by premsagarmeda** [Project-MONAI/MONAILabel#1738] [Q&A]
Hi,

I am facing problem to run auto segmentation in OHIF and MONAI Label server.

I have installed MONAI label server on ubuntu in Anaconda3 with Python 3.9

using **pip install monailabel**

I could start Monai with deepedit with our orthac using the below command

monailabel start_server --app apps/radiology --studies https://ourdomain.com/dicom-web --conf models deepedit

Server started successfully.

After opening OHIF using the url below

https://ourdomain.com:8000/ohif/

In this when I start auto segmentation it gets failed with the below error.  

Image ID is Undefined.

To resolve this I have removed cache and also tried uploading cases to orthanc from command storescu and GUI as well, but could not helped these steps to resolve.

I would appreciate it if someone could help me to resolve this issue.

![WhatsApp Image 2024-08-18 at 21 43 50](https://github.com/user-attachments/assets/a6595ce9-0484-4ace-b67b-f25b712b864f)

![WhatsApp Image 2024-08-18 at 21 44 37](https://github.com/user-attachments/assets/72b989e2-a6d6-405b-be89-38056efc4d80)

**Comment by SachidanandAlle** [comment]
Check on the viewer side (browser debug tools) and see what is in the inference request.  Is image id correct in the request?
